lib/roadie/inliner.rb

```diff
@@ -78,7 +82,7 @@ def apply_style_map(style_map)
     end
 
     def apply_element_style(element, builder)
-      element["style"] = [builder.attribute_string, element["style"]].compact.join(";")
+      element['style'] = [builder.attribute_string, element['style']].compact.join(';')
```

Comment:
Why did you change this?

Reply:
Sorry auto formatter preferred single quotes unless interpolation is used. Do you want to introduce rubocop to make coding standards explicit?